.rubocop.yml

```diff
@@ -49,6 +55,18 @@ Naming/PredicateName:
   ForbiddenPrefixes:
     - is_
 
+RSpec/ContextWording:
+  Enabled: true
+  Prefixes:
+    - when
+    - with
+    - without
+    - for
+    - if
+    - as
+    - 'not as'
+    - from
```

Comment:
these were in use and worked in the context they were in.  It made turning on the cop a lot easier.